docs/accounts/org/people.md

```diff
@@ -19,4 +19,10 @@ and click **Add**:
 **Org Owners** have implicit access to all workspaces in the organization, and you cannot revoke their access at the workspace level.  **Members**, on the other hand, are not granted access to any workspaces by default; you may [grant access to a workspace](/pipes/docs/workspaces/people) from the **People** tab on your workspace after they have been added to your organization.
 
 To revoke access from a user, select the options menu button ('three dots' button) to the
-right of the user and click **Remove**. 
\ No newline at end of file
+right of the user and click **Remove**. 
+
+## Last Activity
+
+The **Last Activity** column represents the most recent date on which an activity was performed by a user in an organization. This timestamp helps track the latest user engagement, offering valuable insights into activity trends and system usage within the organization. The following actions currently count as user activity for an organization:
```

Comment:
To be honest, I think your previous one was better - It was vague, but I dont like the bullets here, and its incomplete anyway. Let's use the previous one for now. I feel like we need to rework this whole page at some point anyway.